migrations/sqls/20250630125251-returns-id-down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE returns DROP COLUMN id;
```

Comment:
On any migration, we usually include the schema name to prevent confusion when it is run across various environments. Plus, the previous team managed to pick a protected name for both the schema and table. This is why, when writing ad-hoc queries, we always have to quote `returns`.

```suggestion
ALTER TABLE "returns"."returns" DROP COLUMN id;
```